src/middlewared/middlewared/job.py

```diff
@@ -577,6 +577,7 @@ def start_logging(self):
         if self.logs_path is not None:
             fd = self.logs_fd
             os.makedirs(LOGS_DIR, exist_ok=True)
+            os.chmod(LOGS_DIR, 0o700)
```

Comment:
`os.makedirs` has `mode` argument.

Reply:
Yeah, but that only sets mode when dir is freshly created. It doesn't reset mode.

Reply:
@anodos325 what is the case when it's not freshly created? It is wiped on middleware startup.